data/tutorials/gs_03_ocaml_on_windows.md

```diff
@@ -9,12 +9,49 @@ date: 2021-05-27T21:07:30-00:00
 
 # OCaml on Windows
 
-Full support for OCaml on Windows is actively being worked on and a workable
-environment can already be achieved today. A gentle reminder that if you do not
-need Windows binaries, then a more stable option is to use WSL2. This is
-described on the [up and running page](/docs/up-and-running).
+There is a new [Diskuv OCaml][DKML] ("DKML") Windows
+installer that we recommend for new users. However while [Diskuv OCaml][DKML] has a modern OCaml 4.12.1 compiler,
+it does not track the latest OCaml compilers. We will officially support Windows as a Tier 1
+platform with a [major release of Opam](#opam-22) in the coming months, and it will be compatible with
+DKML installations.
 
-## opam-repository-mingw
+[DKML]: https://github.com/diskuv/dkml-installer-ocaml#readme
+
+Our guidance is when you want:
+
+* **only to run, not develop, applications**, use [Docker](#docker-images) or [WSL2](#wsl2)
```

Comment:
There are so many commas in this sentence.
 Perhaps we can reword it?
Like: 
**only to run (not develop) applications**, use [Docker](#docker-images) or [WSL2](#wsl2)